app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands, source_file, destination_file = command.split()
```

Comment:
What if I am a bad user and will enter this command?
`cp file1.txt file2.txt file3.txt file4.txt`
Your unpacking will fail because of too many elements